src/app/plugins/controller.plugin.ts

```diff
@@ -6,31 +6,22 @@ export class PluginControl extends Plugin {
   public description: string = 'Controls activating and deactivating plugins.';
   public usage: string = "!controller <activate | deactive> <plugin name>";
   public permission: ChannelType = ChannelType.Admin;
+  public commandPattern: RegExp = /^(deactivate|activate) (?!\s*$).+/;
 
   constructor(public container: IContainer) {
     super();
   }
 
+  public validate(message: IMessage, args: string[]) {
+    return (args && args.length >= 2 && this.commandPattern.test(args.join(' ')));
```

Comment:
no need to override validate if you use the command pattern. Sorry for the conflicting reviews
because of the commandPattern, it takes care of the length too